dev-aws/kafka-shared-msk/billing/transaction-auditor-events.tf

```diff
@@ -12,7 +12,7 @@ resource "kafka_topic" "transactions_auditor_diff_events" {
     "remote.storage.enable" = "true"
     # keep data in primary storage for 2 days
     "local.retention.ms" = "172800000"
-    # keep data for 2 months
+    # keep data for 1.9712525667351128 months
```

Comment:
Maybe it would be worth to round it up a little bit?
It makes no practical difference for users, I think

Reply:
Yea, that would be an option indeed. Let me try and see if I manage something with floats